client/components/inline-support-link/index.jsx

```diff
@@ -71,6 +72,11 @@ class InlineSupportLink extends Component {
 			iconSize,
 		};
 
+		// props.children overrides text prop if both are provided
+		const shouldDisplayChildren = children != null;
+		const shouldDisplayTextPropValue = ! children && showText;
+		const shouldDisplayIcon = ! children && supportPostId && showIcon;
```

Comment:
Can maybe shorten the names here to be `displayChildren`, `displayTextPropValue`, and `displayIcon`